content/influxdb/cloud/get-started.md

```diff
@@ -11,66 +11,130 @@ weight: 2
+1.  Click **Data (Load Data)** > **Buckets** in the navigation menu on the left.
 
     {{< nav-icon "data" >}}
 
 2.  Click **{{< icon "plus" >}} Add Demo Data**, and then select the **Website Monitoring** bucket.
 3.  The Demo Data bucket appears in your list of buckets.
 
-## Select metrics
+## Add sample data
 
-Now that we've added the demo data bucket, we're ready to create our notebook.
+{{% note %}}
+For example purposes, we will be referencing the [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data), but we provide other [sample data sets](/influxdb/cloud/reference/sample-data/#sample-datasets) as well.
+{{% /note %}}
 
-1. In the navigation menu on the left, click **Notebooks**.
+Use [sample data](/influxdb/cloud/reference/sample-data/) to quickly populate InfluxDB with sample time series data. Sample data sets delivered with Cloud are used throughout this documentation site to provide context, best practices, and clear examples for how InfluxDB Cloud features and functions work.
 
-    {{< nav-icon "notebooks" >}}
-2. Click **+Create Notebook**.
-3. Enter a name for your notebook in the **Name this notebook** field.
-4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
-5. In the **Metric Selector** cell, click the **Select a bucket** dropdown and choose **Website Monitoring Bucket**.
+1. Click **Data > Bucket** and click **Create Bucket** to create a bucket and name it **noaa**.
+2. Do one of the following to download sample data: 
+   - [Install community templates](#add-sample-data-with-community-templates) 
+   - [Download using the InfluxData UI](#add-sample-data-with-influxdata-ui)
+
+### Add sample data with community templates 
+
+After installing the template, sample data will be stored in the sample data bucket. 
+
+1. Go to the community templates respository on GitHub and head to our [sample data documentation](/influxdata/community-templates/tree/master/sample-data). 
+2. [Install the template](https://github.com/influxdata/community-templates/tree/master/sample-data#quick-install) with the instructions given. 
+
+### Add sample data in InfluxDB Cloud UI 
+   
+1. Click **Data Explorer** in InfluxDB Cloud's left navigation menu and select your **noaa** bucket. 
+2. Click **Script Editor**. 
+3. Copy the `sample.data()` function listed for the NOAA sample dataset and paste it into the script editor. 
+    ```js
+    import "influxdata/influxdb/sample"
+
+    option task = {
+      name: "Collect NOAA NDBC data"
+      every: 15m,
+    }
+
+    sample.data(set: "noaa")
+      |> to(
+          org: "example-org",
+          bucket: "example-bucket"
+      )
+    ```
+   Click here for more information about [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data), or other [sample data sets](/influxdb/v2.0/reference/sample-data/). 
+4. Click **Submit**. 
```

Comment:
```suggestion
Use the [Flux InfluxDB sample package](/{{< latest "flux" >}}/stdlib/influxdata/influxdb/sample/) to download and write sample data to InfluxDB.
Add the following as an [InfluxDB task](/influxdb/cloud/process-data/manage-tasks/create-task/).

```js
import "influxdata/influxdb/sample"

option task = {
  name: "Collect NOAA NDBC data"
  every: 15m,
}

sample.data(set: "noaa")
  |> to(bucket: "noaa"  )
    ```
For more information about this and other InfluxDB sample datasets, see [InfluxDB sample data](/influxdb/cloud/reference/sample-data/). 
```